components/apimgt/org.wso2.carbon.apimgt.rest.api.gateway.v1/src/main/resources/gateway-api.yaml

```diff
@@ -152,13 +156,17 @@ paths:
           in: query
           type: string
           required: true
-        - name: label
+        - name: version
           description: |
-            Label of the API Gateway
+            version of the API Gateway
```

Comment:
This should be the version of the API

Reply:
https://github.com/wso2/carbon-apimgt/pull/9009/commits/b99a273017722380eaa25c62f86587c522498b9a